### ISSUE TITLE
nanoplot

### ISSUE THREAD

**Issue by rditter** [ubccr/software-layer#228]
NanoPlot
https://github.com/wdecoster/NanoPlot

**Comment by tonykew** [comment]
test build of NanoPlot version 1.33.0 complete - more info to follow

**Comment by dsajdak** [comment]
Hi @rditter!  nanoplot has been published to the `ccrsoft/2023.01` software repository

```
vortex:~$ module spider nanoplot

----------------------------------------------------------------------------------------------------------------
  nanoplot: nanoplot/1.33.0
----------------------------------------------------------------------------------------------------------------
    Description:
      Plotting suite for long read sequencing data and alignments


    You will need to load all module(s) on any one of the lines below before the "nanoplot/1.33.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    Help:

      Description
      ===========
      Plotting suite for long read sequencing data and alignments


      More information
      ================
       - Homepage: https://github.com/wdecoster/NanoPlot

vortex:~$ module load gcc/11.2.0  openmpi/4.1.1 nanoplot/1.33.0
```
